uber/site_sections/emails.py

```diff
@@ -25,10 +25,7 @@ def pending_examples(self, session, subject):
         count = 0
         examples = []
         email = AutomatedEmail.instances[subject]
-        attendees, groups = session.everyone()
-        models = {Attendee: attendees, Group: groups}
-        models.update({model: lister(session) for model, lister in AutomatedEmail.extra_models.items()})
-        for x in models[email.model]:
+        for x in AutomatedEmail.queries[email.model](session):
```

Comment:
A good example of an inefficient query strategy: we were literally loading every row of every table in the database that has automated emails just to get the one table we actually care about!